Crashlytics/UnitTests/FIRCLSRecordTests.m

```diff
@@ -214,6 +222,28 @@ - (void)testRecordSignalFile {
   XCTAssertEqual(adapter.storage.total, 63989469184);
 }
 
+- (void)testProtoReport {
+  FIRCLSRecordAdapter *adapter =
+      [[FIRCLSRecordAdapter alloc] initWithPath:[FIRCLSRecordTests persistedCrashFolder]
+                                withGoogleAppId:@"1:17586535263:ios:83778f4dc7e8a26ef794ea"];
+
+  // Used to prototyping:
+  GDTCORTransport *transport = [[GDTCORTransport alloc] initWithMappingID:@"1206"
+                                                             transformers:nil
+                                                                   target:kGDTCORTargetFLL];
+  GDTCOREvent *event = [transport eventForTransport];
+  event.dataObject = adapter;
+  event.qosTier = GDTCOREventQoSFast;  // Bypass batching and have the event get sent out ASAP
+  [transport sendDataEvent:event];
```

Comment:
@mikehaney24 - I am encountering the following issue when sending the event:
[GoogleDataTransport][I-GDTCOR001006] Assertion failed (/Users/jasonhu/git/FirebasePublic/firebase-ios-sdk/GoogleDataTransport/GDTCORLibrary/GDTCORStorage.m:103): There's no prioritizer registered for the given target. Are you sure you've added the support library for the backend you need?

Seems like I'm missing some configuration?

Reply:
Remember that I told you a couple days ago that FLL support had been temporarily removed? As of a couple hours ago it's been put back, so make sure you're sync'd to HEAD on master. If that fails, then make sure you've recently pod gen'd.

Reply:
Yep, that did it!